app/models/submission.rb

```diff
@@ -251,6 +251,83 @@ def committee_email_list
     list.uniq
   end
 
```

Comment:
I'm wondering if we can refactor these methods into a `SubmissionSchema` class kinda like we do in ScholarSphere.  Then implement the `SubmissionSchema` object in `SolrSubmission`.  `Submission` is getting a bit large and unwieldy.  Putting these methods elsewhere I think would make things cleaner.  I can take a stab at this if you're busy.